app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[1] == command_list[2]:
```

Comment:
You should also check if the first word of the command is `cp`.

Reply:
Thanks. Done.